RibonDAO/core-api
# Feat/prod3 461

# Description

- creates a ClearOldTicketsJob and command
- tests

## Does this pull request close any open issues?

- [#prod3-461](https://ribon.atlassian.net/browse/PROD3-461?atlOrigin=eyJpIjoiZWZhYzE4OTg0ODIxNDZhYTk5OWM5NzVmMGYzMzBiZWMiLCJwIjoiaiJ9)

## Type of change


## How to test

- check for your tickets on your terminal (I used Tickets.pluck(:created_at) to help me) 
- run Tickets::ClearOldTicketsJob on your terminal (for deleting tickets older than 1 month, there is no need to pass any arguments)
- check again
